docs/scripts/i18n.js

```diff
@@ -40,7 +41,7 @@ async function run() {
     });
   };
 
-  traverse([...systemPages, ...basePages, ...materialPages]);
+  traverse([...systemPages, ...basePages, ...materialPages, ...joyPages]);
```

Comment:
Ok, this is the reason why `yarn docs:i18n` wasn't failing before.

Reply:
Aha, gottcha!